src/Middleware/Diagnostics.Abstractions/src/IExceptionHandlerFeature.cs

```diff
@@ -14,5 +16,21 @@ public interface IExceptionHandlerFeature
         /// The error encountered during the original request
         /// </summary>
         Exception Error { get; }
+
+        /// <summary>
+        /// The portion of the request path that identifies the requested resource. The value
+        /// is un-escaped.
+        /// </summary>
+        string? Path => null;
```

Comment:
```suggestion
        string Path => throw new NotSupportedException();
```

Reply:
Shouldn't that be NotImplementedException?

Reply:
It's fairly unusual to see NotImplemented outside of test code. Plus it does seem like NotSupported covers the absence of implementation:

> NotSupportedException indicates that no implementation exists for an invoked method or property.

https://docs.microsoft.com/en-us/dotnet/api/system.notsupportedexception?view=net-5.0#remarks

Reply:
I was about to throw in my support for NotSupportedException because it could just be a supported scenario with an old implementation, but the docs do seem to indicate that NotSupportedException is better for new virtual methods on base types:

> You should also throw a NotSupportedException exception if you've implemented an abstract base class and add a new member to it that must be overridden by derived classes. In that case, making the member abstract causes existing subclasses to fail to load.

https://docs.microsoft.com/en-us/dotnet/api/system.notimplementedexception?view=net-5.